src/esw/imu_driver.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import rospy
+from sensor_msgs.msg import Imu, MagneticField, Temperature
+from geometry_msgs.msg import Quaternion, Vector3
+from std_msgs.msg import Header
+from mrover.msg import CalibrationStatus
+
+import serial
+from serial import SerialException, SerialTimeoutException
+
+
+def main():
+    # publishers for all types of IMU data, queue size is 1 to make sure we don't publish old data
+    imu_pub = rospy.Publisher("imu/data", Imu, queue_size=1)
```

Comment:
so in the case of a string youre giving to a publsher or a subscriber, it actually does make a difference. if you put the slash in front, it makes it a "global" topic, so it ignores any xml groups its in and will always subscribe to exactly that topic. if you dont put a slash (which is the convention we follow) then you could wrap the node in an xml group tag with a certain name and it would append that name to the beginning of the topic name. at least thats my understanding

Reply:
i see, very useful explanation. I kinda put slashes on everything for my drive code oops... is it worth making a new pr just to change that or do u want to do that for me in this pr lol

Reply:
I can fix it

Reply:
done, also added a link to a ros wiki page about it to ROS fundamentals

Reply:
Thank youuu